designs/virtualization/4.7/create-vm-from-template/create-virtual-machine-from-template.md

```diff
@@ -0,0 +1,94 @@
+---
+parent: Virtualization
+version: 4.7
+---
+# Developer Console - Create a Virtual Machine from a template flow
+
+## Background
+
+On the developer console, there is currently no way for a developer to quickly create a virtual machine from a template. Virtual machines can only be created on the administrator console by administrators (cluster admins) and developers. We will be introducing this feature to the developer console with this initiative.
+
+## Goals and objectives
+Giving developers the ability to quickly and seamlessly create virtual machines from templates. This should reduce the perspective switching cost and improve the overall console experience.
+
+## Introducing
+- The ability to create VMs on the developer console
+- A “Virtual Machines” card to the “Add” page
+- Virtual machine template cards in the developer catalog
+- The ability to see a virtual machine template option in the main filter on   the developer catalog page
+
+
+
+## General information
+- A developer can create a virtual machine from a template
+- Only templates with boot sources will be available
+- Templates can not be customized
+- There are 4 types of templates:
+- Red Hat supported
+    - Red Hat provided
+    - User provided
+    - User supported
+- Template filtering mechanism will be according to template type only 
+- The developer can select Flavor size from the following:
+    - Tiny
+    - Small
+    - Medium
+    - Large
+
+## Virtual Machine template card details
+The details on the virtual machine template cards are as follows:
+- Card label (Virtual Machines ) upper right corner
+- Template name (text)
+- Template provider information (text)
+- Virtual Machine details:
+    - Type: 
+    - Flavor:
+    - Storage:
+
+## The general flow
+### The Add screen
+![Add screen](img/screen-1-add-resources.png)
+1. **Add screen** - Developer starts their navigation here. If OpenShift Virtualization is installed, the user will see the Virtual Machines card
+
+### The Virtual Machine screen
+![Add screen](img/screen-2-virtual-machine-templates.png)
```

Comment:
I think we can pull the WIP badge on these

Reply:
nit - the design is showing the first filter as "Red Hat" and should say "Red Hat supported"

Reply:
Fixed this text, it now reads correctly "Red Hat supported"